From 35f87f6674e4d9c5c016586a6ff1d9b3f6c4c021 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Thu, 22 Sep 2022 18:32:23 +0200
Subject: [PATCH 0323/1695] dt-bindings: add binding for the
 stm32mp-provisioning

Add binding to manage the sticky lock values for the
shadow provisioning driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 5a3e7c582d3397c96a41a4e351dfe315ff43e16a dt-bindings: add binding for the stm32mp-provisioning]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Ia2b23ebdf0e110b614833271e79259d75e7f2540
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/267531
---
 .../include/dt-bindings/soc/stm32mp-provisioning.h | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 core/include/dt-bindings/soc/stm32mp-provisioning.h

diff --git a/core/include/dt-bindings/soc/stm32mp-provisioning.h b/core/include/dt-bindings/soc/stm32mp-provisioning.h
new file mode 100644
index 000000000..045dc32a3
--- /dev/null
+++ b/core/include/dt-bindings/soc/stm32mp-provisioning.h
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: GPL-2.0+ OR BSD-3-Clause */
+/*
+ * Copyright (C) 2022, STMicroelectronics - All Rights Reserved
+ */
+
+#ifndef _DT_BINDINGS_STM32MP_PROVISIONING_H
+#define _DT_BINDINGS_STM32MP_PROVISIONING_H
+
+#define STICKY_NO_LOCK		0
+#define STICKY_LOCK_SW		1
+#define STICKY_LOCK_SR		2
+#define STICKY_LOCK_SWSR	3
+
+#endif /* _DT_BINDINGS_STM32MP_PROVISIONING_H */
-- 
2.47.2

